sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -597,6 +597,103 @@ object functions {
     grouping_id((Seq(colName) ++ colNames).map(n => Column(n)) : _*)
   }
 
+  /**
+   * Aggregate function: returns the compact binary representation of the Datasketches
```

Comment:
I believe we are returning the updatable representation now.
Same for the functions below.

Reply:
Good catch - updated all of the descriptions to replace 'compact' with 'updatable'.